### ISSUE TITLE
Remove "All rights reserved." from framework and engine source file license headers.

### ISSUE THREAD

**Issue by zanderso** [flutter/flutter#124429]
This will require updating every source file, and the engine's license checker script.

cc @renaudd @Hixie

**Comment by ditman** [comment]
cc @stuartmorgan we may care about this too?

**Comment by stuartmorgan** [comment]
What's the context here? Is it these repos specifically, or all Flutter code (in which case I should change flutter/packages)?

**Comment by zanderso** [comment]
This is based on guidance from @renaudd, and yes my understanding is that it would apply to anything with the Flutter header on it. I retitled the issue to match.

**Comment by stuartmorgan** [comment]
I can update the packages repo tooling that validates this

Is it only the source files, or also the LICENSE files?

**Comment by Hixie** [comment]
Let's take this slowly. It is not urgent.

Before we change any repositories, let's make a plan for how we're going to change _every_ non-archived flutter org repository, and update appropriate verifications in each repo. I want to make sure that the length of time where we have multiple conflicting licenses is kept to an absolute minimum (like, days, not weeks, certainly not months or years). 

If we're going to do this we should also take the opportunity to standardize on a single copyright year for all our repos too, and standardize on copyright-verification CI to make sure we stay in compliance in all our repos.

**Comment by Hixie** [comment]
> The copyright year for a given copyright notice needs to list the first year that the file/project it is attached to was published.

Right, we would standardize on the earliest date across the entire Flutter project. We've already done that (based on legal advice a few years ago) for several of our repos, but currently they aren't on the same year because it was done on a per-repo basis rather than across the entire project.

**Comment by renaudd** [comment]
On Mon, Apr 10, 2023 at 11:32 AM Ian Hickson ***@***.***>
wrote:

> The copyright year for a given copyright notice needs to list the first
> year that the file/project it is attached to was published.
>
> Right, we would standardize on the earliest date across the entire Flutter
> project. We've already done that (based on legal advice a few years ago)
> for several of our repos, but currently they aren't on the same year
> because it was done on a per-repo basis rather than across the entire
> project.
>
Perfect! OSPO has no objections to that.
